### DISCUSSION TITLE
Connecting Strimzi from On-premise DC

### DISCUSSION THREAD

**Discussion by raghavendra-bhat-maersk** [strimzi/strimzi-kafka-operator#8195] [Q&A]
Hi Team,

I am trying to connect to the Strimzi Kafka cluster using External Ingress from our secured on-premise datacenter.

I am able to connect to the Bootstrap and Broker ingress endpoints with port 443 using Curl, however my Kafka producer/consumer clients are not making successful connection to brokers. Getting below error: 

[2023-03-06 06:16:53,076] ERROR Error when sending message to topic mytest.topic.v1 with key: 1 bytes, value: 1 bytes with error: (org.apache.kafka.clients.producer.internals.ErrorLoggingCallback)
org.apache.kafka.common.errors.TimeoutException: Topic mytest.topic.v1 not present in metadata after 60000 ms.

Could you please suggest if anything specific we need to check in our network side? 
I can confirm broker ingress and port are allowed in our firewall and proxy.

Regards,
Raghav

**Comment by sreejesh123** [comment]
Did you customise to run Kafka bootstrap on 443 ?   Default external ingress port  is on 9094 hence the q.

**Comment by scholzj** [comment]
You will probably need to provide more details:
* The custom resources
* Configuration of your clients
* Logs from the clients
* etc.
